src/lib/y2partitioner/widgets/mkfs_optiondata.rb

```diff
@@ -0,0 +1,465 @@
+
+        {
+          fs:          %i(xfs),
+          widget:      :MkfsCheckBox,
+          label:       N_("Inodes &Aligned"),
+          default:     true,
+          mkfs_option: "-ialign=0",
+          # help text, richtext format
+          help:        N_(
+            "This option is used to specify whether inode allocation is aligned. " \
+            "By default inodes are aligned as this is more efficient than unaligned access. " \
+            "But this option can be used to turn off inode alignment when the filesystem " \
+            "needs to be mountable by an old version of IRIX that does not have the " \
+            "inode alignment feature."
+          )
+        },
+
+        {
+          fs:          %i(vfat),
+          widget:      :MkfsComboBox,
+          label:       N_("FAT &Size"),
+          values:      %w(auto 12 16 32),
+          default:     "auto",
+          mkfs_option: "-F",
+          # help text, richtext format
+          help:        N_(
+            "Specify the size of the file allocation tables entries (12, 16, or 32 bit). " \
+            "If 'auto' is specified, a suitable value is chosen dependng on the file system size. " \
+            "Note that choosing an unsuitable FAT size might result " \
+            "in an error when creating the file system."
+          )
+        },
+
+        {
+          fs:          %i(vfat),
+          widget:      :MkfsComboBox,
+          label:       N_("Number of &FATs"),
+          values:      %w(auto 1 2),
+          default:     "auto",
+          mkfs_option: "-f",
+          # help text, richtext format
+          help:        N_(
+            "Specify the number of file allocation tables. " \
+            "The default is 2."
+          )
+        },
+
+        {
+          fs:          %i(vfat),
+          widget:      :MkfsInputField,
+          label:       N_("Root &Dir Entries"),
+          default:     "auto",
+          validate:    lambda do |x|
+            (x.match?(/^\d+$/) && x.to_i >= 16) || x == "auto"
+          end,
+          mkfs_option: "-r",
+          error:       N_(
+            "The minimum number of entries is 16."
+          ),
+          # help text, richtext format
+          help:        N_(
+            "Select the number of entries available in the root directory. " \
+            "Choose a value that is at least 16, or 'auto'. " \
+            "Note that the value is a lower limit and the actual root directory may be larger."
+          )
+        }
+      ]
+
+      # Remember option data.
+      #
+      # @param option [Hash]
+      #
+      def initialize(option)
+        @value = option
+        textdomain "storage"
+      end
+
+      # Read current option value from file system class.
+      #
+      # @param filesystem [Y2Storage::Filesystems]
+      #
+      # @return [String, Boolean]
+      #
+      def get(filesystem)
+        # current filesystem settings
+        fs_str = filesystem.send program + "_options"
```

Comment:
The SUSE style guide encourages to use something like this here:
```ruby
fs_str = filesystem.send(program + "_options)
```
As much as I personally like omitting parenthesis in method calls, most people coming from other languages can be uttered confused by that practice.

Reply:
And this is not enforced by rubocop?